packages/ui/src/components/alert-dialog.tsx

```diff
@@ -162,7 +163,7 @@ const AlertDialogCancel = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <AlertDialogPrimitive.Cancel
     ref={ref}
-    className={cn(buttonVariants({ variant: 'outline' }), 'mt-2 sm:mt-0', className)}
+    className={cn(buttonVariants({ variant: 'surface', theme: 'muted' }), 'mt-2 sm:mt-0', className)}
```

Comment:
instead of adding button classes here, can we render the actual Button component and put an `asChild` on the `Cancel` copmonent?

Reply:
Internally, it is using the button component. So it is as same as using `asChild` with `Button` component.
<img width="1062" alt="image" src="https://github.com/user-attachments/assets/c401e286-7836-41d4-aae2-b06f3478ee8e" />